### PR TITLE
Add FAQ & update notification about MonPsy

### PR DESCRIPTION
**Pull request by sandrica89**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ services:
       - .env
     environment:
       DATABASE_URL: postgres://santepsy:santepsy@db:5432/santepsy
-      ANNOUNCEMENT: "(Docker-compose variable) Very important announcement."
+      ANNOUNCEMENT: "Vous souhaitez rejoindre le réseau des psychologues partenaires du dispositif général MonParcoursPsy ? <a href=\"https://www.demarches-simplifiees.fr/commencer/monpsy\" target=\"_blank\" rel=\"noopener noreferrer\">Cliquer ici</a><br />MonParcoursPsy concerne toute la population à partir de 3 ans."
```
Comment (desoindx) [inline docker-compose.yml:27]: A changer sur l'env de Scalingo :)

File: frontend/src/PsychologistRouter.jsx [hunk 2]
```diff
@@ -91,23 +90,22 @@ const PsychologistRouter = () => {
       withNotification
     >
       {modal}
-      <Announcement />
```
Comment (desoindx) [inline frontend/src/PsychologistRouter.jsx:94]: Why did you remove it here ?

Comment (sandrica89) [reply frontend/src/PsychologistRouter.jsx:94]: I found that it was too messy in the pages with all the callouts and notifications. It also seems unappropriate to put something out from SPE scope inside the pages where user will enter appointments, download bills, etc.

**Review by desoindx — APPROVED** [review]
